Makefile

```diff
@@ -60,9 +60,9 @@ export KUBECTL ?= ./cluster/kubectl.sh
 KUBECTL ?= ./cluster/kubectl.sh
 OPERATOR_SDK ?= $(GOBIN)/operator-sdk
 
-GINKGO = go run github.com/onsi/ginkgo/v2/ginkgo
-CONTROLLER_GEN = go run sigs.k8s.io/controller-tools/cmd/controller-gen
-OPM = go run -tags=json1 github.com/operator-framework/operator-registry/cmd/opm
+GINKGO = GOFLAGS=-mod=mod go run github.com/onsi/ginkgo/v2/ginkgo@v2.1.4
+CONTROLLER_GEN = GOFLAGS=-mod=mod go run sigs.k8s.io/controller-tools/cmd/controller-gen@v0.8.0
```

Comment:
Could to use the same version of controller-gen as before? Because the newer version removes crd:trivialVersions .
To make this change more clear, I think it would be better to use here the version from go.mod, and only then bump it, which will requre removing the trivialVersions param from makefile